landing_page/app/javascript/components/HelloWorld.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+
+class HelloWorld extends React.Component {
```

Comment:
This is a typescript file but you are not using any typescript properties. It would be good to make this component strongly typed